hard/LC_1638

```diff
@@ -0,0 +1,37 @@
+//APPROACH 2: 1-D, Karan Kumar
+//time : O(n*n)
+//space : O(n) + 26 * O(n)
+
+class Solution {
+    vector<vector<int>> freq;
+public:
+    int numWays(vector<string>& words, string target) {
+        int n = target.size();
+        int m = words.front().size();
+        long long mod = 1e9 + 7;
```

Comment:
write constant variable name in UPPER_CASE also make it cost 
`const long long MOD = 1e9 + 7;`